microsoft-365/compliance/customer-key-manage.md

```diff
@@ -163,6 +163,13 @@ The output from this cmdlet includes:
 
   - **Rolling:** A key roll is in progress. If the key for the geo is rolling, you'll also be shown information on what percentage of sites have completed the key roll operation so that you can monitor progress.
 
+## Unassign DEP from a mailbox
+
+You can unassign a Customer Key DEP from a mailbox by assigning $NULL DEP to the mailbox. Doing so will unassign the currently assigned Customer Key DEP and reencrypt the mailbox using the default Microsoft managed keys DEP. You can not unassign the Microsoft managed keys DEP. You can reassign another Customer Key DEP to the same mailbox.
```

Comment:
```suggestion
You can unassign a Customer Key DEP from a mailbox by assigning $NULL DEP to the mailbox. Doing so will unassign the currently assigned Customer Key DEP and re-encrypt the mailbox using the default Microsoft managed keys DEP. You can not unassign the Microsoft managed keys DEP. You can reassign another Customer Key DEP to the same mailbox.
```

Reply:
Needs a bit of an Acrolinx check. I will work on this one in the private repo tonight.